vector/build.gradle

```diff
@@ -139,6 +134,7 @@ dependencies {
     api project(":matrix-sdk-android")
     implementation project(":matrix-sdk-android-flow")
     implementation project(":library:jsonviewer")
+    implementation project(":library:strings")
```

Comment:
Maybe rename the module to `ui-strings` as per `ui-styles`. I am not sure from where this convention comes from. Any idea @ganfra ?

Reply:
will rename :+1: